src/test/cbmc/DynTrait/vtable_size_align_drop_fail.rs

```diff
@@ -13,6 +13,7 @@
 #![allow(deprecated)]
 
 use std::intrinsics::size_of;
+use std::ptr::drop_in_place;
```

Comment:
Why do you need this?

Reply:
`drop_in_place` is used in the test and the test will not compile without it. I copied this use statement from the nonfailing version of the test.